src/sdk/namespace_cache.js

```diff
@@ -47,8 +57,9 @@ class NamespaceCache {
     /////////////////
 
     async list_objects(params, object_sdk) {
-        // TODO listing from cache only for deevelopment
-        return this.namespace_nb.list_objects(params, object_sdk);
+
+        return params.get_from_cache ? this.namespace_nb.list_objects(params, object_sdk) :
```

Comment:
where are this param will come from? doesn't it suppose to be part of s3_get_bucket?

Reply:
Right, I see that @guptsanj added it to s3_get_bucket in #6130